webrender/src/resource_cache.rs

```diff
@@ -116,6 +116,40 @@ struct ImageResource {
     dirty_rect: Option<DeviceUintRect>
 }
 
+struct ImageTemplates {
```

Comment:
I don't understand. How does this change from the borrowck point of view if you just do `type ImageTemplates = HashMap<...>`?